sqlalchemy_cockroachdb/base.py

```diff
@@ -148,12 +148,20 @@ def initialize(self, connection):
         # The crdb_internal.increment_feature_counter only exists in v21.1 of CRDB
         # and onwards.
         if self._is_v211plus and not self.disable_cockroachdb_telemetry:
-            version = pkg_resources.require("sqlalchemy-cockroachdb")[0].version
-            telemetry_query = (
-                "SELECT crdb_internal.increment_feature_counter"
-                + f"('SQLAlchemy {version}')"
+            adapter_version = pkg_resources.require("sqlalchemy-cockroachdb")[0].version
+            sqlalchemy_version = pkg_resources.require("sqlalchemy")[0].version
+            telemetry_query =  "SELECT crdb_internal.increment_feature_counter(:val)"
+            connection.execute(
+                text(telemetry_query).bindparams(
+                    val=f'sqlalchemy-cockroachdb {adapter_version}'
+                )
+            )
+            connection.execute(
+                text(telemetry_query).bindparams(
+                    val=f'sqlalchemy {sqlalchemy_version}'
+                )
             )
-            connection.execute(text(telemetry_query))
+
```

Comment:
The above could also be done with, e.g.,

```python
            connection.execute(
                text(telemetry_query), dict(val=f"SQLAlchemy {sqlalchemy_version}")
            )
```

Nothing wrong with using `.bindparams()` _per se_, but `.execute(text_statement, dict_of_values)` is a more common usage pattern.

Reply:
thank you! i spent some time and couldn't figure out the idiomatic way easily, so was hoping you could help me :)